src/mobile/radio-button/index.ts

```diff
@@ -155,11 +155,13 @@ let exportMobileRadioButton;
             value="${item.value !== undefined ? item.value : ""}"
             aria-invalid="${this.error !== ""}"
             aria-required="${this.requiredIcon}"
-            ?disabled="${this.disabled}"
+            ?disabled="${this.disabled || item.disabled}"
             @change="${this._handleChangeInput}"
           />
           <label
-            class="kuc-mobile-radio-button__group__select-menu__item__label"
+            class="kuc-mobile-radio-button__group__select-menu__item__label ${item.disabled
+              ? " kuc-mobile-radio-button__group__select-menu__item__label--disabled"
+              : ""}"
```

Comment:
It's not good to set this CSS class for label.
Because when an item is disabled, the space around the item is not grey.

Please try to set this class for `kuc-mobile-radio-button__group__select-menu__item`.